---a/net/minecraft/network/play/client/C14PacketTabComplete.java
+++b/net/minecraft/network/play/client/C14PacketTabComplete.java
@@ -1,71 +1,65 @@
 package net.minecraft.network.play.client;
 
 import java.io.IOException;
+
 import net.minecraft.network.INetHandler;
 import net.minecraft.network.Packet;
 import net.minecraft.network.PacketBuffer;
 import net.minecraft.network.play.INetHandlerPlayServer;
 import net.minecraft.util.BlockPos;
+
 import org.apache.commons.lang3.StringUtils;
 
-public class C14PacketTabComplete implements Packet
-{
+public class C14PacketTabComplete implements Packet {
+
     private String message;
     private BlockPos targetBlock;
     private static final String __OBFID = "CL_00001346";
 
-    public C14PacketTabComplete() {}
+    public C14PacketTabComplete() {
+    }
 
-    public C14PacketTabComplete(String msg)
-    {
-        this(msg, (BlockPos)null);
+    public C14PacketTabComplete(String msg) {
+        this(msg, (BlockPos) null);
     }
 
-    public C14PacketTabComplete(String msg, BlockPos target)
-    {
+    public C14PacketTabComplete(String msg, BlockPos target) {
         this.message = msg;
         this.targetBlock = target;
     }
 
-    public void readPacketData(PacketBuffer buf) throws IOException
-    {
+    public void readPacketData(PacketBuffer buf) throws IOException {
         this.message = buf.readStringFromBuffer(32767);
         boolean flag = buf.readBoolean();
 
-        if (flag)
-        {
+        if (flag) {
             this.targetBlock = buf.readBlockPos();
         }
     }
 
-    public void writePacketData(PacketBuffer buf) throws IOException
-    {
+    public void writePacketData(PacketBuffer buf) throws IOException {
         buf.writeString(StringUtils.substring(this.message, 0, 32767));
         boolean flag = this.targetBlock != null;
         buf.writeBoolean(flag);
 
-        if (flag)
-        {
+        if (flag) {
             buf.writeBlockPos(this.targetBlock);
         }
     }
 
-    public void processPacket(INetHandlerPlayServer handler)
-    {
+    public void processPacket(INetHandlerPlayServer handler) {
         handler.processTabComplete(this);
     }
 
-    public String getMessage()
-    {
+    public String getMessage() {
         return this.message;
     }
 
-    public BlockPos getTargetBlock()
-    {
+    public BlockPos getTargetBlock() {
         return this.targetBlock;
     }
 
-    public void processPacket(INetHandler handler)
-    {
-        this.processPacket((INetHandlerPlayServer)handler);
+    public void processPacket(INetHandler handler) {
+        this.processPacket((INetHandlerPlayServer) handler);
     }
-}
+}
